tests/virt_autotest/setup_dns_service.pm

```diff
@@ -40,7 +46,7 @@ sub post_fail_hook {
     save_screenshot;
     script_run("cat /var/lib/named/123.168.192.zone");
     save_screenshot;
-    script_run("mv /etc/resolv.conf.orig /etc/resolv.conf; mv /etc/named.conf.orig /etc/named.conf");
+    script_run("mv /etc/resolv.conf.orig /etc/resolv.conf; mv /etc/named.conf.orig /etc/named.conf; mv /etc/ssh/ssh_config.orig /etc/ssh/ssh_config; mv /etc/dhcpd.conf.orig /etc/dhcpd.conf");
```

Comment:
IMPORTANT: please split the total changes in this PR at least to 2 commits, one for enhancement to dns, one for guest integration support.

Reply:
The contents in this PR are all integral parts. They should belong to one. I hope you can understand.

Reply:
The contents in this PR are all integral parts. They should belong to one. I hope you can understand. Any enhancement servers for the completeness of the functionality of this PR, and for reviewer's bettering understanding of what has been done here.

Reply:
I totally agreen to put them in one PR. BUt I do not understand why changes to DNS as well as the code change to integrate new feature tests are necessarily to be in one *commit*. The benefit of splitting them into two commits can help when for example some revert work needed, and we can keep one commit while whatever actions needed can be played to the other commit.

Reply:
One commit is suggested to focus on one purpose.